docs/users.md

```diff
@@ -113,12 +113,18 @@ Save the configuration and then activate the plugin (button on top).
 
 This plugin allows GitHub users to authenticate to Xen-Orchestra.
 
-The first time a user signs in, XO will create a new XO user with the same identifier, without any permissions.
+The first time a user signs in, XO will create a new XO user with the same identifier (i.e. Github name), with "user" permissions. An existing admin will need to apply the appropriate permissions for your environment.
```

Comment:
- *GitHub* capitalization
- you can use `*word*` to highlight a word in Markdown

Reply:
I have updated this now